src/main/java/io/conduit/Transformations.java

```diff
@@ -54,20 +54,13 @@ public static final Record.Builder fromKafkaSource(SourceRecord sourceRecord) {
         return Record.newBuilder()
                 .setKey(getKey(sourceRecord))
                 .setPayload(getPayload(sourceRecord))
-                .setCreatedAt(currentTimestamp());
+                // we need nanoseconds here
+                .putMetadata(OpenCdcMetadata.CREATED_AT, String.valueOf(System.currentTimeMillis() * 1_000_000));
```

Comment:
Hmm, this sounds like it should be `OpenCdcMetadata.READ_AT`.

Reply:
Good one. :+1: